.github/workflows/ci.yml

```diff
@@ -124,7 +158,11 @@ jobs:
   linux-aws-lc-fips:
     runs-on: ubuntu-22.04 # latest
```

Comment:
you changed all the other ubuntu to be 24.04, but this one is still 22.04 Did you miss this one by accident? If you left it 22.04 on purpose, leave a comment why